src/Sarif/VersionConstants.cs

```diff
@@ -8,4 +8,4 @@ public static class VersionConstants
         public const string SchemaVersionAsPublishedToSchemaStoreOrg = "2.1.0-rtm.5";
         public const string StableSarifVersion = "2.1.0";
     }
-}
+}
```

Comment:
we should have a new line in the end. With dotnet-format, we are checking that as well
```bash
Run dotnet-format --folder --check --exclude .\src\Sarif\Autogenerated\
  Formatting code files in workspace 'D:\a\sarif-sdk\sarif-sdk'.
  sarif-sdk\src\Sarif\VersionConstants.cs(11,2): Fix final newline.
  Formatted code file 'D:\a\sarif-sdk\sarif-sdk\src\Sarif\VersionConstants.cs'.
  Format complete in 5983ms.
```

Reply:
It should be the following, I think. This will still work in the pipeline (where the bld and packages dirs don't exist)

Run dotnet-format --folder --check --exclude .\src\Sarif\Autogenerated\ --exclude .\src\packages\ --exclude .\bld\

---
In reply to: [510433793](https://github.com/microsoft/sarif-sdk/pull/2125#discussion_r510433793) [](ancestors = 510433793)

Reply:
no problem, i can create a new pr adding those exclusions. The point here is that we need to add a new line in the end for this file :)

---
In reply to: [510439559](https://github.com/microsoft/sarif-sdk/pull/2125#discussion_r510439559) [](ancestors = 510439559,510433793)